Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -61,6 +61,13 @@ public extension StorageType {
         return firstObject(ofType: OrderNote.self, matching: predicate)
     }
 
+    /// Retrieves the Stored OrderCount.
+    ///
+    func loadOrderCount(siteID: Int) -> OrderCount? {
```

Comment:
could this be unit tested in this or a future PR?

Reply:
I don't think we've never tested these methods, but let's start now! I'll add a test class for this extensions with a test for this method

Reply:
It turns out there might be a reason for that. As far as I can tell, the setup would be a bit too complicated. 

I don't like to defer things like these, but in this case... I've added a ticket for it, will try to get to it after the whole Orders Badge is merged:

#1144